administrator/language/de-DE/plg_api-authentication_basic.ini

```diff
@@ -0,0 +1,8 @@
+; Joomla! German Translation
+; Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.
+; Copyright (C) Translation 2008 - 2019 J!German (www.jgerman.de)
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_API-AUTHENTICATION_BASIC="API Authentifizierung - Basisauthentifizierung"
+PLG_AUTH_AUTHENTICATION_BASIC_XML_DESCRIPTION="Wird verwendet, um die Basisauthentifizierung für Web Services in Joomla zu ermöglichen."
```

Comment:
Vereinheitlichung: für Web Services oder von Web Services.
Oder ist das eine von Web Services genutzt und das andere wird Web Services zur Verfügung gestellt?

Reply:
Ich würde sagen dieses spezielle Plugin wird `für` Web Services gebraucht. `Vom` Web Service müsste dann in den die anderen Plugins nutzen.